cw_infrastructure/cw_infrastructure/clearwater_options.py

```diff
@@ -17,16 +17,22 @@
 
 def get_options():
     """Set up the list of options to be validated"""
+
+    # Setup the SAS server validator depending on whether signaling namespace is to be used
+    sas_server_validator = vlds.resolvable_ip_or_domain_name_validator 
+    if (os.environ.get('sas_use_signaling_interface') == 'Y'):
+        sas_server_validator = vlds.run_in_sig_ns(vlds.resolvable_ip_or_domain_name_validator)
```

Comment:
You don't need the brackets round the if.

I think:

```python
sas_server_validator = vlds.resolvable_ip_or_domain_name_validator

if os.environ.get('sas_use_signaling_interface') == 'Y':
        sas_server_validator = vlds.run_in_sig_ns(sas_server_validator)
```

would be better.

I think we should probably also add a validator for sas_use_signaling_interface as well - probably:

```python
Option('sas_use_signaling_interface', Option.OPTIONAL, vlds.yes_no_validator),
```

and an appropriate validator - e.g.:

```python
def yes_no_validator(name, value):
    if value == 'Y' or value == 'N':
       return OK
   else:
        error(name, "{} is not a valid value - should either be Y or N".format(value))
        return ERROR
```

Reply:
Sounds like a good idea.